apps/meeting/src/components/DeviceSelection/CameraDevices/BackgroundReplacementDropdown.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import React, { ChangeEvent, useState } from 'react';
+import {
+  useBackgroundReplacement,
+  FormField,
+  Select,
+  useVideoInputs,
+} from 'amazon-chime-sdk-component-library-react';
+import { BackgroundImageEncoding, createColorBlob, createImageBlob } from '../../../utils/image';
+import { ReplacementDropdownOptionType, ReplacementOptions } from '../../../types/index';
+import { useAppState } from '../../../providers/AppStateProvider';
+
+interface Props {
+  /* Title for the dropdown, defaults to `Background Replacement Dropdown` */
+  label?: string;
+}
+
+export const BackgroundReplacementDropdown: React.FC<Props> = ({
+  label = 'Background Replacement Dropdown',
+}) => {
+  const { selectedDevice } = useVideoInputs();
+  const { backgroundReplacementOption, setBackgroundReplacementOption } = useAppState();
+  const { isBackgroundReplacementSupported, changeBackgroundReplacementImage } = useBackgroundReplacement();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const options: ReplacementDropdownOptionType[] = [
+    {
+      label: ReplacementOptions.Blue,
+      value: isBackgroundReplacementSupported === undefined || isBackgroundReplacementSupported === false ? 'Background Replacement not supported' :ReplacementOptions.Blue,
+    },
+    {
+      label: ReplacementOptions.Beach,
+      value: isBackgroundReplacementSupported === undefined || isBackgroundReplacementSupported === false ? 'Background Replacement not supported' :ReplacementOptions.Beach,
+    },
+  ];
```

Comment:
1. Could we merge `ReplacementDropdownOptionType` and `ReplacementOptions ` into one type? Something like using `Record<K, V>`.
- https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

2. Could we handle isSupported and placeholder in another place? The flow could be something like this:

    1. check if isSupported
    2. if true, iterate through `options` and renders dropdown options
    3. if false, iterate through a fallback `options` that renders `Background Replacement not supported`

Reply:
Yeah, we could hide `BackgroundReplacementDropdown` if it is not supported.